src/login.rs

```diff
@@ -197,10 +197,11 @@ async fn password_login(
                 if sms {
                     match sms_phone {
                         None => {
-                            resp = client.request_sms().await.expect("无法请求短信验证码");
+                            panic!("未绑定手机号，无法使用短信验证码登录");
                         }
                         Some(sms_phone) => {
                             // TODO: test
+                            client.request_sms().await.expect("无法请求短信验证码");
```

Comment:
```suggestion
                            let sms_phone = sms_phone.clone();
                            resp = client.request_sms().await.expect("无法请求短信验证码");
                            match resp {
                                LoginResponse::DeviceLocked(..) => {}
                                _ => continue,
                            }
```
我打算这样处理 `resp`，不过不确定是否必要，而且我也不确定 `request_sms()` 成功的结果就是 `LoginResponse::DeviceLocked` （之后我再验证一下）。